agri-store-react/src/Components/Context/WishlistPageContext.js

```diff
@@ -0,0 +1,44 @@
+import axios from "axios";
+import React, { createContext } from "react";
+import { useProductContext } from "./ProductContextReducer";
+export const wishlistContext = createContext();
+function WishlistPageContext({ children }) {
+  const { state, dispatch } = useProductContext();
+  const { wish } = state;
+
```

Comment:
creating a context file and importing everything from a single file would look more clean!

Reply:
Can you please explain a bit , with respect to the file / folders i have made